### ISSUE TITLE
Trying to find abstract model class' table instead of model's own ctable

### ISSUE THREAD

**Issue by bmedici** [lowjoel/activerecord-userstamp#7]
Hello, this construction doesn't seem to work well with activerecord-userstamp  :

```
class ActiveAccounting < ActiveRecord::Base
end
class Workday < ActiveAccounting
  stampable
end
```

Leads to:

```
ActiveRecord::StatementInvalid - Mysql2::Error: Table 'bmconseil_dev.active_accountings' doesn't exist: SHOW FULL FIELDS FROM `active_accountings`
```

Of course, active_accountings table does not exist nor needed to. 
Any idea what I'm doing wrong or how to tell the module to only focus on model and not its inheritance chain ?

Backtrace: https://gist.github.com/bmedici/79d1f034e939b910e7d3858dc95ac59b

Thanks !

**Comment by lowjoel** [comment]
Yikes, I'll look at it when I can (which probably won't be at least for another 2 weeks, sorry :sob:)

I'm not entirely sure if this is the same problem as always automatically inferring the creator/updater associations, though, which is caused by the callbacks being called the moment a class is inherited.

**Comment by lowjoel** [comment]
@bmedici can you have a look at the PR in #12, see if that solves this too?

**Comment by lowjoel** [comment]
Should be fixed by #12.
